.travis.yml

```diff
@@ -1,16 +1,24 @@
 language: cpp
```

Comment:
A minor one: Use Imperative form for the commit title, e.g: `Add minor fixes ...`